spec/integration/schema_parser_spec.rb

```diff
@@ -77,7 +117,7 @@ def to_dto
           end
         end
         it 'raises a InternalArgumentError valid data' do
-          request = build_mock_request(params: {}, headers: {}, body: valid_body)
+          request = build_mock_request(params: {}, headers: {}, body: invalid_jsonapi_body)
```

Comment:
`valid_body` was renamed to `valid_jsonapi_body` – so why not use it here as well? (just a minor thing to mention as it wouldn't change the outcome of this spec since the provided schema is invalid anyway)